app/services/discord.server.ts

```diff
@@ -1,23 +1,60 @@
-import type { DiscordGuildMember } from "~/models/discordProfile.server";
+import redisClient from "~/services/redis.server";
 
 export async function getDiscordGuildMembershipByProfileId(
```

Comment:
`getDiscordGuildMembershipByProfileId`

the function name implies it will return a `DiscordGuildMember` but it returns a `boolean`

Reply:
Great suggestion, done